src/App.tsx

```diff
@@ -1,7 +1,26 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    pressed: false,
+    key: null,
+  };
+
+  componentDidMount = () => {
+    document.addEventListener('keyup', (event) => {
+      this.setState({ pressed: true, key: event.key });
+    });
```

Comment:
According to checklist 2, we have to remove this event listener, when component will unmount from page.